rhel8/profiles/e8.profile

```diff
@@ -0,0 +1,138 @@
+documentation_complete: true
+
+title: 'Australian Cyber Security Centre (ACSC) Essential Eight System Hardening for Red Hat Enterprise Linux 8'
```

Comment:
@shaneboulden can you remove 'for Red Hat Enterprise Linux 8' and add it to the description?

Reply:
Done, and shortened the title to reflect other profiles